performant_pagination/pagination.py

```diff
@@ -103,7 +103,14 @@ def _object_to_token(self, obj):
             value = obj._meta.get_field(pieces[-1]).value_to_string(obj)
 
         # return our value
-        return b64encode(value)
+        if isinstance(value, text_type):
+            value = value.encode("utf-8")
+
+        value = b64encode(value)
+
+        if not isinstance(value, text_type):
+            value = value.decode("utf-8")
+        return value
```

Comment:
To double check here. Is there a case when it's not of `text_type`? I believe it always should be, since it comes from `.value_to_string`. If so I think it's sufficent do just do:

```python
value = b64encode(value.encode('utf-8'))
```